CHANGELOG.md

```diff
@@ -2,6 +2,33 @@
 
 ## master
 
+## 0.8.0 (2018-10-01)
+
+- [PR [#87](https://github.com/palkan/logidze/pull/87)] Adding debounce time to avoid spamming changelog creation
```

Comment:
Please, add your Github handle here (and in the bottom of the document)

Reply:
thanks, it should be done